addons/settings/fnc_readFile.sqf

```diff
@@ -0,0 +1,32 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_settings_fnc_readFile
+
+Description:
+    Reads and applies settings from a file.
+    Only considered as public API from a static settings addon!
```

Comment:
Unsure how to make this clear. In theory it is public API because people use it in their own static settings PBOs, but we also don't want anyone to use outside of that I assume.